privacy-protections/storage-blocking/helpers/idb-wrapper.js

```diff
@@ -0,0 +1,72 @@
+// https://raw.githubusercontent.com/msandrini/minimal-indexed-db/master/index.js
```

Comment:
just a indexeddb wrapper. I thought I can use native for the simple thing I'm doing, but indexeddb API is so bad that I gave up.